cluster/playbook/roles/common/tasks/kubernetes.yaml

```diff
@@ -10,12 +10,21 @@
     state: present
 
 - name: Install Kubeadm, kubelet and kubectl
+  when: kubernetes_version is not defined
   apt:
     pkg:
       - kubelet
       - kubeadm
       - kubectl
 
+- name: Install Kubeadm, kubelet and kubectl of version {{kubernetes_version}}
```

Comment:
we can merge these two section into one using default variable [filter](https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#defaulting-undefined-variables).

If we have two sections, later on if we introduce another package, we need to mention it in two different place